client/src/components/Shared/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import Button from '@mui/material/Button';
+import Tooltip, { TooltipProps, tooltipClasses } from '@mui/material/Tooltip';
+import Typography from '@mui/material/Typography';
+
+const HtmlTooltip = styled(({ className, ...props }: TooltipProps) => (
```

Comment:
If you're doing styling here -- which, btw, is more idiomatic for MUI so that's usually how I do it at least -- you don't need `Tooltip.scss`

Reply:
made this change!